internal/common/agent_util_windows.go

```diff
@@ -102,6 +102,8 @@ func RunShellScript(path string, args ...string) (string, error) {
 	shellArgs = append(shellArgs, path)
 	shellArgs = append(shellArgs, args...)
 
+	log.Printf("running %v", shellArgs)
```

Comment:
super mild nit - it might make more sense to join the args into a string than try to print the slice. 
```suggestion
	log.Printf("running %s", strings.Join(" ", shellArgs))
```